docs/GetStarted/getting_started_python.md

```diff
@@ -0,0 +1,61 @@
+# Getting Started with Python
+
+IREE has Python bindings geared towards lower level compiler interop that are
+not intended to be a public API, and integration with Python frontends such as
+TensorFlow.
+
+We do not yet provide a pip package for easy installation, so to use IREE's
+Python bindings you must build from source.
+
+## Prerequisites
+
+You should already have IREE cloned and building on your machine. See the other
+[getting started guides](.) for instructions.
+
+> Note:<br>
+> &nbsp;&nbsp;&nbsp;&nbsp;Support is best with Bazel.
+> For CMake (excluding TensorFlow), set the `IREE_BUILD_PYTHON_BINDINGS` option.
```

Comment:
fyi @marbre 

could document CMake Python support in here (particularly how to run tests on `bindings/python/...`?)